### PR TITLE
Add note on using commas in alias descriptions

### PR DESCRIPTION
**Pull request by StanczakDominik**
On my installation, using commas in docstring descriptions causes a `error: unsupported operand type(s) for %: 'list' and 'dict'` error. Wrapping the description in a string solves that.

### CODE REVIEW

File: pubs/config/spec.py [hunk 1]
```diff
@@ -99,6 +99,11 @@
 # command = !pubs list -k | wc -l
 # description = lists number of pubs in repo
 
+# To use commas in the description, wrap them in a "" string. For example:
+# [[[hellocount]]]
+# command = !pubs list -k | wc -l; echo "hello!"
```
Comment (omangin) [inline pubs/config/spec.py:104]: Minor comment here: for the sake of conciseness, I would suggest keeping only the first and last line of your change (no need for a new fake section and a new fake comment).

**Review by omangin — CHANGES_REQUESTED** [review]
Thanks @StanczakDominik for taking the time to edit the config documentation, and sorry that it took us so long to answer. See my minor comment below. Other than that, it is good to merge.

File: pubs/config/spec.py (hunk 1 above)
Comment (StanczakDominik) [reply pubs/config/spec.py]: Done :)

**Review by omangin — APPROVED** [review]
Great! Thanks @StanczakDominik!

**Comment by omangin** [comment]
Note: you might have to rebase on master once #245 is merged for the tests to pass.

**Comment by benureau** [comment]
Rebased and merged to main. Apologies for the delay.

**Pull request closed without merging**